includes/wf_crm_webform_postprocess.inc

```diff
@@ -1562,16 +1562,24 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
       $billing_fields['credit_card_number'] = FALSE;
       $billing_fields['cvv2'] = FALSE;
     }
-    if (!empty($_POST['bank_account_type'])) {
+    if (isset($_POST['bank_account_type'])) {
```

Comment:
I had followed the lead for the Stripe implementation: that's why it's an `isset`
`if (!empty($_POST['stripe_token'])) {`